syntax = "proto3";
package Imperative_KEYPAD;
option optimize_for = LITE_RUNTIME;

message BUTTON {
  uint32 uid = 1;
  string function = 2;
};
